Neos.Flow/Classes/Mvc/View/JsonView.php

```diff
@@ -12,11 +12,13 @@
  * source code.
  */
 
+use GuzzleHttp\Psr7\Response;
+use GuzzleHttp\Psr7\Utils;
 use Neos\Flow\Annotations as Flow;
-use Neos\Flow\Mvc\Controller\ControllerContext;
 use Neos\Flow\Persistence\PersistenceManagerInterface;
 use Neos\Utility\ObjectAccess;
 use Neos\Utility\TypeHandling;
+use Psr\Http\Message\ResponseInterface;
```

Comment:
deprecate

Reply:
Now that we deprecated the json view ... what is our answer to code like this? :O

```php
/**
 * @var array
 */
protected $viewFormatToObjectNameMap = [
    'html' => FusionView::class,
    'json' => JsonView::class,
];
```

do we need a new simple json view? :P

cc @kitsunet